docs/sphinx/_ext/dagster-sphinx/dagster_sphinx/configurable.py

```diff
@@ -93,7 +93,7 @@ def config_field_to_lines(field, name=None) -> List[str]:
             if isinstance(val, dict):
                 ls = json.dumps(val, indent=4).split("\n")
                 lines.append("")
-                lines.append("    .. collapse:: Default Value:")
```

Comment:
I need to revert this, this ends up creating a very long doc for some APIs. I think the proper way to deal with this is to move this logic into the json builder rather than to hijack lines, since we want to output this for old docs, but not for new mdx-docs.